en/docs/references/alerting.md

```diff
@@ -0,0 +1,61 @@
+
+#### How to troubleshoot Out of Memory error in the Observability view
+
+When you click the Check in Portal button you will be redirected to the Observe page of the component in the Choreo console.
+
+![OOM Troubleshooting](../assets/img/alerting/oom-troubleshooting.png){.cInlineImage-full}
+
+**1** shows the time bin where the Out of Memory error occurred will be highlighted for you and the logs that occurred during that time will be given within the scrollable Logs column. It will also contain the counts of how many such logs occurred.
+
+![Log bins](../assets/img/alerting/log-bins.png){.cInlineImage-half}
+
+The link given in the alert email will select a 1hour time range around the time in which the error occurred (i.e. error time +/- 30mins). This is shown in the diagram as **2**. The **Diagnostic View** is divided into 5 similar sized time bins. In this case, the size of each bin will be 12 mins. This is shown by **3**.
+
+![Diagnostic view](../assets/img/alerting/diagnostic-view-values.png){.cInlineImage-half}
+
+When you hover over the graph, you can see the Memory usage and it growing steadily and reaching the maximum value. This will cause the application to restart and it will log the Out of Memory error.
+
+#### Selecting custom time ranges
+
+If the time range that is selected automatically for you is not enough, you can select the time range that you desire and debug further. You can do this by clicking on the Time selection drop down bar shown by 1.
+
+![Custom Time range](../assets/img/alerting/custom-time-range.png){.cInlineImage-half}
+
+Then you can select the **Custom** option, this will automatically add the time range that is already given by the alert email. You can modify them to see the **Diagnostic View** for that  specific time range.
+
+For more information on how to use the Observability view for Root cause analysis check out the following [page](https://wso2.com/choreo/docs/observability/root-cause-analysis/). 
+
+### Application Error Alert
+
+This alert is triggered when you use the **“log:printError()”** function in your component and it logs an error. We consider that your component must have faced some serious problem and send you an email to notify you about this. The sample email will look like this. The email will provide details about the application and the **Check in Portal** link to view this log along with the request that caused this. You can also get support from WSO2 to troubleshoot this issue by using the **Contact us** link provided in the email.
+
+![Application Error email](../assets/img/alerting/application-error-email.png){.cInlineImage-full}
+
+#### How to troubleshoot Application error in the Observability view
```

Comment:
> 
Is it necessary to mention what we are troubleshooting? 
`Troubleshooting Application error`
Because we also have a similar heading at the top of the article
 `How to troubleshoot Out of Memory error in the Observability view`

Reply:
Since this is a subheading of the error type, it is understood. However, if it is better to mention what we are trouble shooting, we can have it as "Troubleshoot application error" and avoid present continuous tense in the heading.

Reply:
Ack